data/fixtures/recorded/actions/copySecondToken.yml

```diff
@@ -0,0 +1,36 @@
+languageId: plaintext
+command:
+  version: 3
+  spokenForm: copy second token
+  action: {name: copyToClipboard}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: ordinalScope
+          scopeType: {type: token}
+          start: 1
+          length: 1
+initialState:
+  documentContents: |
+
+    const value = "Hello world";
+  selections:
+    - anchor: {line: 1, character: 0}
+      active: {line: 1, character: 0}
+  clipboard: value
```

Comment:
can we remove this or change it to something else? note that the value afterwards is the same, so we're not really checking that anything gets copied

Reply:
Indeed we can remove it, it was a typo. Thanks